src/main/java/seedu/address/model/person/Resume.java

```diff
@@ -74,8 +78,18 @@ private static boolean isPdf(byte[] data) {
                     data[data.length - 5] == 0x45 && // E
```

Comment:
Might want to link the reference to validation of pdf files :smiley:

Reply:
I believe he gets it from here: https://stackoverflow.com/questions/941813/how-can-i-determine-if-a-file-is-a-pdf-file

Besides, I might extract this part out to a utility class or use PDF library to validate when I update on UI during the next milestone

Reply:
I think Yee Chin's approach is better. My way of checking is quite stupid = =